README.md

```diff
@@ -508,6 +508,53 @@ await authnClient.CancelTransactionStateAsync(transactionStateOptions);
 
 ```
 
+## Send information via User Agent
+
+Some scenarios require you to send additional information in the request's user agent. The following methods allow you to provide your user agent information via the method options:
```

Comment:
Rather than "some scenarios", describe them and link to the docs (device fingerprinting examples in AuthN API docs, in this case)